cedar-policy-cli/Cargo.toml

```diff
@@ -13,6 +13,7 @@ repository = "https://github.com/cedar-policy/cedar"
 [dependencies]
 cedar-policy = { version = "=3.0.0", path = "../cedar-policy" }
 cedar-policy-formatter = { version = "=3.0.0", path = "../cedar-policy-formatter" }
+cedar-policy-validator = { version = "=3.0.0", path = "../cedar-policy-validator" }
```

Comment:
why does CLI need a direct dependency on Validator?  should we just expose the relevant Validator functionality in `cedar-policy` instead?